jdat_notebooks/NIRISS_WFSS_postpipeline/00. Optimal extraction.ipynb

```diff
@@ -4,18 +4,28 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "## 00 Optimal extraction of NIRISS WFSS spectrum;\n",
+    "## 00 Optimal Extraction of NIRISS WFSS Spectrum\n",
```

Comment:
```suggestion
    "## Part 0: Optimal Extraction of NIRISS WFSS Spectrum\n",
```
(Goes with the similar changes elsewhere in this series)

Note that this one is a bit awkwardly named "part 0", but it's probably best to have the number match the file name?  If that doesn't bother you then all of these can just be Part 1/2/3/4 instead of Part 0/1/2/3